PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -483,7 +482,7 @@ INT_PTR CALLBACK Shortcut::run_dlgProc(UINT Message, WPARAM wParam, LPARAM)
 // return true if one of CommandShortcuts is deleted. Otherwise false.
 void Accelerator::updateShortcuts() 
 {
-	vector<int> incrFindAccIds;
+	vector<UINT> incrFindAccIds;

```

Comment:
For signed and unsigned comparison go to line 520 below:
```C#
// Special extra handling for shortcuts shared by Incremental Find dialog
if (std::find(incrFindAccIds.begin(), incrFindAccIds.end(), shortcuts[i].getID()) != incrFindAccIds.end())
```
shortcuts[i].getID() is unsigned int and  incrFindAccIds was int